app/controllers/api/providers_controller.rb

```diff
@@ -50,6 +51,20 @@ def delete_resource(type, id = nil, _data = nil)
       end
     end
 
+    def update_custom_attributes_resource(_type, _id, data)
+      provider = ExtManagementSystem.includes(:custom_attributes).find_by_name(data.delete("name"))
+      data["custom_attributes"].each do |attribute|
+        if CustomAttribute::ALLOWED_API_FILED_TYPES.include? attribute["field_type"]
+          attribute["value"] = attribute["field_type"].safe_constantize.parse(attribute["value"])
```

Comment:
@alongoldboim is this safe enough? I would feel more comfortable to have a table of supported types that we allow.